yandexcloud/auth.py

```diff
@@ -0,0 +1,15 @@
+from yandex.cloud.iam.v1.iam_token_service_pb2_grpc import IamTokenServiceStub
+from yandexcloud import SDK
+from yandexcloud._auth_fabric import get_auth_token_requester
+
+
+def get_iam_token_by_metadata():
```

Comment:
Better support `metadata_addr` param as in https://github.com/yandex-cloud/python-sdk/blob/master/yandexcloud/_auth_fabric.py#L50. Just make it None by default. So if one want's to set it, it will be possible